test/e2e/common.go

```diff
@@ -207,21 +214,14 @@ func NewPVC(namespace, name, storageClassName, diskSize string, accessModel core
 	}
 }
 
-func NewService(namespace, serviceName, portName string, portNumber, targetPort int32, labels map[string]string) *corev1.Service {
+func newService(namespace, serviceName string, servicePorts []corev1.ServicePort, labels map[string]string) *corev1.Service {
```

Comment:
Please keep use `NewService`, so that other cases from another project can reuse this function